web/src/store/reducers/lineage.ts

```diff
@@ -5,26 +5,31 @@ import {
   FETCH_LINEAGE_SUCCESS,
   RESET_LINEAGE,
   SET_BOTTOM_BAR_HEIGHT,
+  SET_LINEAGE_GRAPH_DEPTH,
   SET_SELECTED_NODE
 } from '../actionCreators/actionTypes'
 import { HEADER_HEIGHT } from '../../helpers/theme'
 import { LineageGraph } from '../../types/api'
 import { Nullable } from '../../types/util/Nullable'
-import { setBottomBarHeight, setSelectedNode } from '../actionCreators'
+import { setBottomBarHeight, setLineageGraphDepth, setSelectedNode } from '../actionCreators'
 
 export interface ILineageState {
   lineage: LineageGraph
   selectedNode: Nullable<string>
   bottomBarHeight: number
+  depth: number
 }
 
 const initialState: ILineageState = {
   lineage: { graph: [] },
   selectedNode: null,
-  bottomBarHeight: (window.innerHeight - HEADER_HEIGHT) / 3
+  bottomBarHeight: (window.innerHeight - HEADER_HEIGHT) / 3,
+  depth: 1
```

Comment:
Can we make this default to a larger number that would render the whole demo graph at least?

Reply:
Good point, updated it to `5`. Trying to bias towards a lower value as, at least in my experience playing around with the demo, it feels less clunky to slowly increase from a lower number to "zoom out" vs. walking down from a higher number to "zoom in".